src/aui-scheduler/js/aui-scheduler-base.js

```diff
@@ -1447,6 +1466,19 @@ var SchedulerBase = A.Component.create({
 
                 instance.syncEventsUI();
             }
+        },
+
+        /**
+         * Set the `showToolbar` property - i.e., defines if the scheduler
+         * header will be displayed.
+         *
+         * @method _uiSetShowToolbar
+         * @protected
+         */
+        _uiSetShowToolbar: function() {
```

Comment:
To be fair, the called function should be protected as well, but it was not created that way and I'd rather not change it.